keyless/pepper/service/src/main.rs

```diff
@@ -85,6 +86,12 @@ async fn main() {
         Duration::from_secs(10),
     );
 
+    jwk::start_jwk_refresh_loop(
+        &get_sample_iss(),
```

Comment:
@zjma, this isolates the impact of a malicious JSON file to only the test `iss`. Isn't this fine?

(Trying to address your concern.)